site.hs

```diff
@@ -20,12 +28,21 @@ main = hakyll $ do
                 >>= loadAndApplyTemplate "templates/default.html" blogCtx
                 >>= replaceIndexLinks
 
-    match allPosts $ do
-        route $ setExtension "" `composeRoutes` indexedRoute
-        compile $ pandocCompiler
-            >>= loadAndApplyTemplate "templates/post.html"    postCtx
-            >>= loadAndApplyTemplate "templates/default.html" postCtx
-            >>= replaceIndexLinks
+    create ["feed.xml"] $ do
+        route idRoute
+        compile $ do
+            posts <- recentFirst =<< loadAllSnapshots allPosts "content"
+            let feedConfig = FeedConfiguration
+                    siteTitle           -- title
+                    "Gordon Fontenot"   -- description
+                    "Gordon Fontenot"   -- author name
+                    "gordon@fonten.io"  -- author email
+                    siteHost            -- root
+
+            renderAtom feedConfig feedItemCtx posts
+                >>= replaceIndexURLs siteHost
+                >>= repairExternalURLs siteHost
+                >>= replaceRelativeURLs siteHost
```

Comment:
I've tried that and it didn't look as nice to me. I'll try it again in a different PR.